### ISSUE TITLE
Scanning won't start with latest version

### ISSUE THREAD

**Issue by jkytomaki** [jareff-g/ALT-Scann8#44]
With current master, I can't get scanning to start anymore. Camera real time preview works, but when I click on "Start" button, nothing happens, and I get the following output on the console.

```
2024-01-05 00:30:25,423 [INFO] Camera stopped
[7:22:12.644468461] [6169]  INFO Camera camera.cpp:1033 configuring streams: (0) 2028x1520-BGR888 (1) 2028x1520-SGBRG12_CSI2P
[7:22:12.647651699] [6116]  INFO RPI vc4.cpp:565 Sensor: /base/soc/i2c0mux/i2c@1/imx477@1a - Selected sensor format: 2028x1520-SGBRG12_1X12 - Selected unicam format: 2028x1520-pGCC
2024-01-05 00:30:25,432 [INFO] Configuration successful!
2024-01-05 00:30:25,493 [INFO] Camera started
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.9/tkinter/__init__.py", line 1892, in __call__
    return self.func(*args)
  File "/home/janne/ALT-Scann8/ALT-Scann8-UserInterface.py", line 2047, in start_scan
    button_status_change_except(Start_btn, ScanOngoing)
  File "/home/janne/ALT-Scann8/ALT-Scann8-UserInterface.py", line 1031, in button_status_change_except
    if except_button != Free_btn:
NameError: name 'Free_btn' is not defined
2024-01-05 00:30:30,493 [WARNING] More than 6 sec. since last command: Forcing new frame event (frame 0).
2024-01-05 00:30:36,503 [WARNING] More than 6 sec. since last command: Forcing new frame event (frame 0).
2024-01-05 00:30:42,513 [WARNING] More than 6 sec. since last command: Forcing new frame event (frame 0).
2024-01-05 00:30:48,520 [WARNING] More than 6 sec. since last command: Forcing new frame event (frame 0).
2024-01-05 00:30:54,525 [WARNING] More than 6 sec. since last command: Forcing new frame event (frame 0).
```

I'm also running the latest Arduino code on the nano.

**Comment by jareff-g** [comment]
Issue should be fixed in current version (1.8.18).
It was caused by a recent change where the 'unlock reels' button was moved to the experimental area so that, if experimental mode was not allowed (via -x command line) a reference to the button somewhere else was causing the error. That reference is now protected by checking before if experimental is enabled.
However, I cannot access my scanner now, so please let me know if ti works before I close the issue.

**Comment by jkytomaki** [comment]
Tested it and can't reproduce it anymore. Closing the issue.
